storage/setup.py

```diff
@@ -53,7 +53,7 @@
 REQUIREMENTS = [
     'google-cloud-core >= 0.28.0, < 0.29dev',
     'google-api-core >= 0.1.1, < 0.2.0dev',
-    'google-auth >= 1.0.0',
+    'google-auth >= 1.2.0',
```

Comment:
@jonparrott Should we also update the requirement in `api_core/setup.py`, `bigquery/setup.py` or `spanner/setup.py`?

Reply:
@dhermes This pin is to get the `AnonymousCredentials` feature, which they aren't likely to need.

Reply:
@tseaver I'm aware, thanks. This question doesn't need to block the PR, but is still worth noting.